### DISCUSSION TITLE
lens not loaded exception: bad access - error after blendmode patching

### DISCUSSION THREAD

**Discussion by Leraner** [ptrumpis/snap-camera-server#142] [Q&A]
<img width="800" alt="image" src="https://github.com/ptrumpis/snap-camera-server/assets/53614700/57df4060-d9d2-4721-8ea1-5b8982cff7cb">

After uploading lens to snap-camera-server I got an error unknown blendmode. After that I changed blendmode to 13 in repacked lens and upload it to snap-camera-server, then I got error lens not loaded exception: bad access.

Patch - yes
Assets - yes
Unofficial update - yes

Lens which in snap camera work perfect, but my custom lens throw me "lens not loaded exception: bad access."

What could be the error?

**Comment by DonSudak** [comment]
I have the same problem 
tried to edit lens and reup docker 
any solves?

**Comment by Pampam000** [comment]
I’m facing the same issue. I tried modifying Lens and restarting Docker, but it didn’t work. Are there any solutions?

**Comment by torianos** [comment]
Same problem, help pls 🥲
